src/lib/logger.ts

```diff
@@ -179,15 +190,17 @@ export class Logger {
     Logger.log(event, parameters);
   }
 
-  public static logDocumentEvent(event: LogEventName, document: DocumentModelType) {
+  public static logDocumentEvent(event: LogEventName, document: DocumentModelType,
+    teacherNetworkInfo?: ITeacherNetworkInfo) {
     const parameters = {
       documentUid: document.uid,
       documentKey: document.key,
       documentType: document.type,
       documentTitle: document.title || "",
       documentProperties: document.properties?.toJSON() || {},
       documentVisibility: document.visibility,
-      documentChanges: document.changeCount
+      documentChanges: document.changeCount,
+      ...(teacherNetworkInfo || {})
```

Comment:
```suggestion
      ...teacherNetworkInfo
```
Do you need the `|| {}`? I was under the impression that `...undefined` would be acceptable here.